lib/rubocop/cop/layout/redundant_line_break.rb

```diff
@@ -0,0 +1,95 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Layout
+      # This cop checks whether certain expressions, e.g. method calls, that could fit
+      # completely on a single line, are broken up into multiple lines unnecessarily.
+      #
+      # @example
```

Comment:
You should probably metnion some other type of cases here - like blocks, method chains, and `\` separated lines, so it's more obvious what the cop can tackle.

Reply:
Good idea.